src/amber/cli/templates/app/README.md.ecr

```diff
@@ -1,59 +1,34 @@
 # <%= @name %>
 
-<%= @name %> is a new project using [Amber Framework](https://amberframework.org/)
+This project is powered by [Amber Framework](https://amberframework.org/).
 
 ## Installation
 
-Install shard dependencies.
-
-```
-shards install
-```
-
-Configure the `config/environments/development.yml` and set the `database_url` with your credentials to your database.
-
-Then:
-
-```
-amber db create migrate
-```
+1. [Install required dependencies](https://github.com/amberframework/online-docs/blob/master/getting-started/quick-start.md#install-crystal-and-amber)
+2. Run `shards install`
 
 ## Usage
 
-### Development
-
-To run amber development server:
+To run amber server in a development enviroment:
 
 ```
+amber db create migrate
 amber watch
 ```
 
-### Production
-
-To setup `AMBER_ENV`:
+To build and run a production release:
```

Comment:
To build and run a production release: 

- Add an environment variable `AMBER_ENV` with a value of `production`

WDYGT?